kit/azure/bootstrap/template/platform/platform.hcl

```diff
@@ -1,10 +1,11 @@
 locals {
-  # define shared configuration here that's included by all terragrunt configurations in this locals 
+  # define shared configuration here that's included by all terragrunt configurations in this locals
   platform = yamldecode(regex("^---([\\s\\S]*)\\n---\\n[\\s\\S]*$", file(".//README.md"))[0])
 
   # if we use terraform_state_storage, it will generate this file here to provide backend configuration
   terraform_state_config_file_path = "${get_parent_terragrunt_dir()}/tfstates-config.yml"
   tfstateconfig                    = try(yamldecode(file(local.terraform_state_config_file_path)), null)
+  cloudfoundation_name             = yamldecode(regex("^---([\\s\\S]*)\\n---\\n[\\s\\S]*$", file("../..//README.md"))[0])
```

Comment:
d: for consistency, I'd strongly recommend putting this on L4 so we have 

```
platform = yamldecode....
foundation = yamldecode...
```

That way I see "aha, we're parsing the collie config files for platform and foundation here.

Reply:
Ok than i will changing to foundation.